pkg/controllers/vpc/VPCNetworkConfiguration_handler.go

```diff
@@ -21,21 +23,27 @@ type VPCNetworkConfigurationHandler struct {
 	vpcService *vpc.VPCService
 }
 
+func (h *VPCNetworkConfigurationHandler) buildNetworkConfigInfo(vpcConfigCR v1alpha1.VPCNetworkConfiguration) vpc.VPCNetworkConfigInfo {
+	ninfo := vpc.VPCNetworkConfigInfo{
+		Name:                           vpcConfigCR.Name,
+		DefaultGatewayPath:             vpcConfigCR.Spec.DefaultGatewayPath,
+		EdgeClusterPath:                vpcConfigCR.Spec.EdgeClusterPath,
+		NsxtProject:                    vpcConfigCR.Spec.NSXTProject,
+		ExternalIPv4Blocks:             vpcConfigCR.Spec.ExternalIPv4Blocks,
+		PrivateIPv4CIDRs:               vpcConfigCR.Spec.PrivateIPv4CIDRs,
+		DefaultIPv4SubnetSize:          vpcConfigCR.Spec.DefaultIPv4SubnetSize,
+		DefaultSubnetAccessMode:        vpcConfigCR.Spec.DefaultSubnetAccessMode,
+		LoadBalancerVPCEndpointEnabled: true,
+	}
+	return ninfo
+}
+
 func (h *VPCNetworkConfigurationHandler) Create(e event.CreateEvent, _ workqueue.RateLimitingInterface) {
 	vpcConfigCR := e.Object.(*v1alpha1.VPCNetworkConfiguration)
 	vname := vpcConfigCR.GetName()
-	ninfo := &vpc.VPCNetworkConfigInfo{
-		Name:                    vname,
-		DefaultGatewayPath:      vpcConfigCR.Spec.DefaultGatewayPath,
-		EdgeClusterPath:         vpcConfigCR.Spec.EdgeClusterPath,
-		NsxtProject:             vpcConfigCR.Spec.NSXTProject,
-		ExternalIPv4Blocks:      vpcConfigCR.Spec.ExternalIPv4Blocks,
-		PrivateIPv4CIDRs:        vpcConfigCR.Spec.PrivateIPv4CIDRs,
-		DefaultIPv4SubnetSize:   vpcConfigCR.Spec.DefaultIPv4SubnetSize,
-		DefaultSubnetAccessMode: vpcConfigCR.Spec.DefaultSubnetAccessMode,
-	}
-	h.vpcService.RegisterVPCNetworkConfig(vname, *ninfo)
-	log.Info("create vpc network config object and insert to storage", "vpc", ninfo)
+	ninfo := h.buildNetworkConfigInfo(*vpcConfigCR)
+	log.Info("create network config and update to store", "NetworkConfigInfo", ninfo)
```

Comment:
log level 2?

Reply:
I'd perfer to leave this message Info level, because this is the only log for handling network config CR creation event.